src/openrct2/ride/ride_ratings.c

```diff
@@ -621,21 +622,25 @@ static void ride_ratings_calculate_value(rct_ride *ride)
 
 	sint32 monthsOld = gDateMonthsElapsed - ride->build_date;
 
-	// New ride reward
-	if (monthsOld <= 12) {
-		value += 10;
-		if (monthsOld <= 4)
-			value += 20;
-	}
+	if (!gCheatsDisableRideValueAging) {
+		// New ride reward
+		if (monthsOld <= 12) {
```

Comment:
I know this is not really what your PR is about but could we refactor the whole of this into some sort of table so that you look up the amount to add to value then look up the multiplier.